packages/web3-hooks/base/src/useHiddenAddressConfig.ts

```diff
@@ -11,6 +11,7 @@ type StorageValueV3 = { hiddenAddresses?: Record<string, string[]> }
 type CurrentStorageValue = StorageValueV3
 type StorageValue = StorageValueV1 | StorageValueV2 | StorageValueV3
 
+type T = UseQueryResult
```

Comment:
do you need to add this one? if a file already imports `UseQueryResult` and uses it somewhere, it's highly likely that it doesn't need this mark.